resources/idlharness.js

```diff
@@ -2194,7 +2217,10 @@ IdlInterface.prototype.test_members = function()
         case "iterable":
             this.add_iterable_members(member);
             break;
-        // TODO: add setlike and maplike handling.
+        case "maplike":
+            this.add_maplike_members(member);
+            break;
+         // TODO: add setlike and maplike handling.
```

Comment:
This comment is now wrong.

Reply:
OK I will change.May I know whether the implementation of maplike is correct?